packages/ssx-react/src/ssx.tsx

```diff
@@ -32,12 +38,21 @@ export interface SSXContextInterface {
   ssx: SSX | undefined;
   /** SSX Instance loading state. */
   ssxLoaded: boolean;
+  /** NextAuth Sign In function. */
+  signIn?: (
+    options?: SignInOptions,
+    authorizationParams?: SignInAuthorizationParams
+  ) => Promise<any>;
+  /** NextAuth Sign Out function. */
+  signOut?: (options?: SignOutParams) => Promise<any>;
 }
 
 /** Default, uninitialized context. */
 const defaultContext: SSXContextInterface = {
   ssx: undefined,
   ssxLoaded: false,
+  signIn: undefined,
+  signOut: undefined,
```

Comment:
AN alternative approach, where the signIn and signOut nextauth functions are preserved and call the appropriate SSX stuff